modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -53,75 +58,92 @@ $ vim bmh.yaml
 +
 [source,yaml]
 ----
+---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: openshift-worker-<num>-network-config-secret <1> <2>  
+ name: openshift-worker-<num>-network-config-secret <1> <2>
 type: Opaque
 stringData:
  nmstate: |
-   routes:
-     config:
-     - destination: 0.0.0.0/0
-       next-hop-address: 192.168.123.1
-       next-hop-interface: enp0s4
-   dns-resolver:
-     config:
-       server:
-       - 192.168.123.1
-   interfaces:
-   - name: enp0s4
-     state: up
-     ipv4:
-       address:
-       - ip: 192.168.123.16
-         prefix-length: 24
-       enabled: true
-       dhcp: false
+  routes: <3>
+   config:
+   - destination: 0.0.0.0/0
+     next-hop-address: <next_hop_address>
+     next-hop-interface: <next_hop_interface>
+  dns-resolver:
+   config:
+    server:
+    - <dns_server_ip_address>
+  interfaces:
+  - name: <nic_interface>
+    state: up
+    ipv4:
+     address:
+     - ip: <ip_address>
+       prefix-length: <cidr>
+     enabled: true
+     dhcp: false
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: openshift-worker-<num>-bmc-secret <2> 
+  name: openshift-worker-<num>-bmc-secret <2>
   namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64-of-uid> <3>
-  password: <base64-of-pwd> <4>
+  username: <base64_of_uid> <4>
+  password: <base64_of_pwd> <5>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
   name: openshift-worker-<num> <2>
   namespace: openshift-machine-api
 spec:
-  online: true
-  bootMACAddress: <NIC1-mac-address> <5>
+  online: True
+  bootMACAddress: <nic1_mac_address> <6>
   bmc:
-    address: <protocol>://<bmc-ip> <6>
+    address: <protocol>://<bmc_url> <7>
     credentialsName: openshift-worker-<num>-bmc-secret <2>
-    disableCertificateVerification: true <7>
-    username: <bmc-username> <8>
-    password: <bmc-password> <9>
-  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <10> <2>
+    disableCertificateVerification: True <8>
+    username: <bmc_username> <9>
+    password: <bmc_password> <10>
+  rootDeviceHints:
+    deviceName: <root_device_hint> <11>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <12>
 ----
++
 <1> Optional: To configure network for a newly created node, specify the name of the secret that contains the network configuration. Follow the `nmstate` syntax to define the network configuration for your node.
```

Comment:
The entire secret (lines 65-86) and the reference to it (like 114) are optional, not just the name.